apps/vc-api/src/vc-api/exchanges/exchange.service.ts

```diff
@@ -44,6 +45,10 @@ export class ExchangeService {
   ) {}
 
   public async createExchange(exchangeDefinitionDto: ExchangeDefinitionDto) {
+    if (!isNil(await this.exchangeRepository.findOne(exchangeDefinitionDto.exchangeId))) {
```

Comment:
```suggestion
    if (!await this.exchangeRepository.findOne(exchangeDefinitionDto.exchangeId)) {
```

Why we can't simplify it?